### PR TITLE
Remove require userId and creditCardId

### PR DESCRIPTION
**Pull request by ashkan18**
Update MP's schema to match latest changes in `exchange.graphql` https://github.com/artsy/exchange/pull/64

In this PR we address two change:
- Remove required `userId` when creating an order by `createOrder`.
- Remove required `creditCardId` when submitting and order.

Rest of the changes in `exchange.graphql` (adding `setShippment` and `setPayment`) will be added in a separate PR.

### CODE REVIEW

**Comment by ashkan18** [comment]
Self-merging this to get ready for walkthrough today, will address any comments in follow up PR :)

File: src/schema/ecommerce/submit_order_mutation.js [hunk 1]
```diff
@@ -14,16 +14,12 @@ const SubmitOrderInputType = new GraphQLInputObjectType({
...
-      type: new GraphQLNonNull(GraphQLString),
-      description: "Credit card ID",
-    },
   },
 })
 
 export const SubmitOrderMutation = mutationWithClientMutationId({
   name: "SubmitOrder",
-  decription: "Submitss an order with payment",
+  decription: "Submitss an order",
```
Comment (sweir27) [inline src/schema/ecommerce/submit_order_mutation.js]: hehe `Submitss` --> `Submits`

File: src/data/exchange.graphql [hunk 2]
```diff
@@ -211,11 +248,47 @@ type RejectOrderPayload {
   order: Order
 }
 
+# Autogenerated input type of SetPayment
+input SetPaymentInput {
```
Comment (sweir27) [inline src/data/exchange.graphql]: These seem tangential to what the PR is doing?

Comment (ashkan18) [reply src/data/exchange.graphql]: this is auto-generated latest Exchange GraphQL schema copied over here, will follow up with PRs for adding setPayment and setShipping.

Comment (sweir27) [reply src/data/exchange.graphql]: a-ha! Makes sense.

File: src/schema/ecommerce/submit_order_mutation.js (hunk 1 above)
Comment (ashkan18) [reply src/schema/ecommerce/submit_order_mutation.js]: also `decription` -> `description` :)

**Pull request merged**